python/nano/example/pytorch/inference_pipeline/resnet/README.md

```diff
@@ -25,6 +25,7 @@ pip install --upgrade numpy==1.21.6
 Initialize environment variables with script `bigdl-nano-init` installed with bigdl-nano.
 ```
 source bigdl-nano-init
+unset KMP_AFFINITY
```

Comment:
we should not have the user to manually manage this @MeouSker77 @TheaperDeng

Reply:
It seems the default value of `KMP_AFFINITY` in `bigdl-nano-init` is wrong, now its default value will cause program use only one core , so here unset it to use more cores. I'll fix its default value to use all cores by default

Reply:
discuss with @MeouSker77 and he will fix this bug so that this line can be removed.

Reply:
This could be a really tricky one. KMP_AFFINITY default value has some conflict with onnxruntime's core resource control (which makes it behave strangely). We cannot unset or reset this sys variable once the user start their python script. Some solution we have used in training is creating a new process(with different KMP_AFFINITY) to handle the work, while it is not reasonable as well since creating a process will cost 1-10ms and very unfriendly to low latency requirement. 

One possible solution is:
1. make `KMP_AFFINITY` default value be None(that is not setting this value in `bigdl-nano-init`)
2. Only set this value during multi-process training by creating new processes (since this var does not give much postive affect for inferencing or single process training)

What do you think? @jason-dai @MeouSker77

Reply:
Now the default value of `KMP_AFFINITY` is `granularity=fine,compact,1,0`. In my test, if remove the `compact` option, i.e., set `KMP_AFFINITY="fine,1,0"` can avoid the conflict with onnxruntime, so it's also a possible solution.

Here is the explanation of the `compact` option:
![image](https://user-images.githubusercontent.com/34214113/190106185-20a37a37-195a-4ce1-81e2-70adf94e033e.png)

Shall we take this solution? @jason-dai @TheaperDeng 

By the way, the multi-processes training will set `KMP_AFFINITY` for sub-processes automatically, it won't be affected by this default value.

Reply:
> Now the default value of `KMP_AFFINITY` is `granularity=fine,compact,1,0`. In my test, if remove the `compact` option, i.e., set `KMP_AFFINITY` to `granularity=fine,1,0` can avoid the conflict with onnxruntime, so it's also a possible solution.
> 
> Here is the explanation of the `compact` option: ![image](https://user-images.githubusercontent.com/34214113/190106185-20a37a37-195a-4ce1-81e2-70adf94e033e.png)
> 
> Shall we take this solution? @jason-dai @TheaperDeng
> 
> By the way, the multi-processes training will set `KMP_AFFINITY` for sub-processes automatically, it won't be affected by this default value.

I think that should be OK.

Reply:
Looks good; need to test with our performance suit.